android/src/main/java/io/ably/flutter/plugin/AblyMessageCodec.java

```diff
@@ -136,6 +136,20 @@ private void WriteJsonElement(ByteArrayOutputStream stream, JsonElement value){
         }
     }
 
+    /**
+     * Converts Map to JsonObject, ArrayList to JsonArray and
+     * returns null if these 2 types are a no-match
+     * */
+    static JsonElement readValueAsJsonElement(final Object object) {
+        final Gson gson = new Gson();
+        if(object instanceof Map) {
```

Comment:
Please leave a space after `if` statements - i.e. in this case:

```suggestion
        if (object instanceof Map) {
```

Reply:
Same as https://github.com/ably/ably-flutter/pull/23#discussion_r471921091

052467d338e6fd8a37e3474c03b355e0ec1b8fdc